<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTInterfaceClass</class>
 <widget class="QMainWindow" name="QTInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTInterface</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:red;</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>16777215</width>
      <height>16777215</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QGroupBox" name="groupBox_login">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">subcontrol-position: right;</string>
      </property>
      <property name="title">
       <string>Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>196</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_username">
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_password">
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>77</width>
         <height>56</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_login">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_signUp">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Sign up</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_signUp">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>SignUp</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_username_signup">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password_signup">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_day">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>04</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>06</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>07</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>08</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>09</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>26</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>27</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>31</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_month">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>04</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>06</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>07</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>08</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>09</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_year">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>2022</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2021</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2020</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2019</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2018</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2017</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2016</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2015</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2014</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2013</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2012</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2011</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2010</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2009</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2008</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2007</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2006</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2005</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2004</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2003</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2002</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2001</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1999</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1998</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1997</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1996</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1995</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1994</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1993</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1992</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1991</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1990</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1989</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1988</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1987</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1986</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1985</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1984</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1983</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1982</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1981</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1980</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_male">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_female">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>76</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_signup">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Birthdate</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>280</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_mainPage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>551</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Main Page</string>
      </property>
      <widget class="QPushButton" name="pushButton_myList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>My list</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_likedMovies">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Liked</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_watchedMovies">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Watched</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_profile">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>40</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>50</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTV_Shows">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>TV Shows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMovies">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Movies</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWatchlist">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>235</y>
      <width>135</width>
      <height>138</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionAccount"/>
    <addaction name="actionLiked_Movies"/>
    <addaction name="actionHistory"/>
    <addaction name="actionSogn_Out"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuTV_Shows"/>
   <addaction name="menuMovies"/>
   <addaction name="menuWatchlist"/>
   <addaction name="menuProfile"/>
  </widget>
  <action name="actionhuyeba">
   <property name="text">
    <string>huyeba</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionSign_Out">
   <property name="text">
    <string>Sign Out</string>
   </property>
  </action>
  <action name="actionSign_Out_2">
   <property name="text">
    <string>Sign Out</string>
   </property>
  </action>
  <action name="actionLiked_Movies">
   <property name="text">
    <string>Liked Movies</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionSogn_Out">
   <property name="text">
    <string>Sign Out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTInterface.qrc"/>
 </resources>
 <connections/>
</ui>
